packages/cdk-blue-green-container-deployment/src/ecs-deployment-group.ts

```diff
@@ -154,11 +162,13 @@ export class EcsDeploymentGroup extends Resource implements IEcsDeploymentGroup
         TerminationWaitTimeInMinutes: terminationWaitTime.toMinutes(),
         AutoRollbackOnEvents: autoRollbackOnEvents,
         DeploymentConfigName: this.deploymentConfig.deploymentConfigName,
+        Tags: Lazy.any({ produce: () => this.tags.renderTags() }),
+        ArnForDeploymentGroup: this.arnForDeploymentGroup(this.application.applicationName, deploymentGroupName),
       },
     });
 
     this.deploymentGroupName = ecsDeploymentGroup.ref;
```

Comment:
Move lines 170 and 171 above the custom resource, and use the property `deploymentGroupName` instead.